functions/zram.bats

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bats 
+
+load zram
+
+@test "installation-zram" {
```

Comment:
If the test need to be skipped because of the x86 CI environment can't performed it use the `skip`  function defined by bats-core framework: https://github.com/bats-core/bats-core#skip-easily-skip-tests